demos/paired_mrus_brain/README.md

```diff
@@ -5,23 +5,22 @@
 **NOTE**
 
 Please read the
-[DeepReg Demo Disclaimer](https://github.com/DeepRegNet/DeepReg/blob/master/demos/README.md).
+[DeepReg Demo Disclaimer](https://deepreg.readthedocs.io/en/325-improve-contributing-pages/demo/introduction.html#demo-disclaimer).
```

Comment:
```
[DeepReg Demo Disclaimer](introduction.html#demo-disclaimer).
```

Sorry we might have some error earlier the disclaimer link should be like above

Reply:
@mathpluscode I just fixed the link and the branch is up-to-date with the master branch.